mlir/utils/jenkins/parameterSweeps.py

```diff
@@ -407,23 +407,32 @@ def main() -> bool:
         help="The build directory of MIOpen",
     )
     args = parser.parse_args()
-    options = Options(debug=args.debug, quiet=args.quiet, xdlops=args.xdlops,
+    options = Options(debug=args.debug, quiet=args.quiet,
         arch=','.join(getArch()), concurrent_tests=args.jobs)
     paths = MIOpenDriver.create_paths(args.mlir_build_dir, args.miopen_build_dir)
 
+    codepath = "none"
```

Comment:
Can we get a command-line option to force the codepath?

(and probably a similar `--rocmlir-gen-flags` as in the main perf script?)